packages/spindle-ui/src/Modal/AppealModal.css

```
@@ -0,0 +1,251 @@
+/*
+ * AppealModal
+ * NOTE: Styles can be overridden with "--AppealModal-*" variables
+*/
+:root {
+  --AppealModal-image-border: 1px inset var(--color-border-low-emphasis);
+  --AppealModal-image-borderRadius: 8px;
+}
+
+.spui-AppealModal {
+  background: var(--color-surface-primary);
+  border: none;
+  border-radius: 20px;
+  /* TODO: replace color with color palette */
+  box-shadow: 0 11px 28px rgba(8, 18, 26, 0.12);
+  padding: 0;
+  width: 400px;
+}
+
+/* stylelint-disable plugin/selector-bem-pattern */
+/* NOTE: set no animations to the dialog-polyfill browsers for stability */
+.spui-AppealModal[open]:not(.spui-AppealModal--closing),
+.spui-AppealModal[open]:not(.spui-AppealModal--closing).spui-AppealModal::backdrop {
+  animation: 0.5s cubic-bezier(0, 0, 0, 1) spui-AppealModal-fade-in;
+}
+
+.spui-AppealModal--closing,
+.spui-AppealModal--closing.spui-AppealModal::backdrop {
+  animation: 0.3s cubic-bezier(0, 0, 0, 1) spui-AppealModal-fade-out;
```

```suggestion
  animation: 0.15s cubic-bezier(0, 0, 0, 1) spui-AppealModal-fade-out;
```
こちらも同じく、durationが変わるかもしれないと思いました